api/tests/integration/infrastructure/repositories/account-recovery-demand-repository_test.js

```diff
@@ -47,38 +50,35 @@ describe('Integration | Infrastructure | Repository | account-recovery-demand-re
       });
 
       context('when demand is unused yet', () => {
-        const email = 'someMail@example.net';
-        let demandId;
 
-        beforeEach(() => {
-          demandId = databaseBuilder.factory.buildAccountRecoveryDemand({ email, temporaryKey, used: false }).id;
+        it('should return the account recovery demand when temporary key not expired', async () => {
```

Comment:
**chipotage**
Pour être exact, il faudrait dire `..and demand exists and temporary key is unique...`
Je proposse
`it('should return the account recovery demand when demand is still valid', async () => {`